qiskit_machine_learning/algorithms/inference/qbayesian.py

```diff
@@ -0,0 +1,335 @@
+# This code is part of a Qiskit project.
+#
+# (C) Copyright IBM 2021, 2023.
```

Comment:
For all this new code that presumably has just been created the earlier date can be removed in such cases. I.e just have `# (C) Copyright IBM 2023.` as the code never existed in the earlier time.

Reply:
Make sense. I will do so.